.github/workflows/check.yml

```
@@ -18,6 +18,23 @@ defaults:
 
 jobs:
+  check-server:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          cache: 'pip'
+      - run: |
+        python -m venv .venv
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 30: could not find expected ':'

(syntax-check)